deeprank2/query.py

```diff
@@ -48,8 +48,10 @@ class Query:
         chain_ids (list[str] | str): the chain identifier(s) of the variant residue or interacting interfaces.
             Note that this does not limit the structure to residues from this/these chain(s).
         pssm_paths (dict[str, str]): the name of the chain(s) (key) and path to the pssm file(s) (value).
-        distance_cutoff (float): the maximum distance between two nodes to generate an edge connecting them.
         targets (dict[str, float]) = Name(s) (key) and target value(s) (value) associated with this query.
+        interaction_radius (float | None): all residues within this radius from the variant residue or interacting interfaces
+            will be included in the graph, irrespective of the chain they are on.
+        max_edge_distance (float | None): the maximum distance between two nodes to generate an edge connecting them.
```

Comment:
I'd suggest `max_nodes_distance` (maximum distance between nodes for having an edge) or `max_distance_edge` (same but making nodes implicit) to avoid confusion from the naming

Reply:
I agree that distance is not the correct terminology. How** about `max_edge_length`?

Reply:
it's a bit tough because "length" and "distance" both have a different meaning than this in [graph theory](https://www.geeksforgeeks.org/graph-measurements-length-distance-diameter-eccentricity-radius-center/)

Reply:
max_edge_length